tests/yam/agama/patch_agama_tests.pm

```diff
@@ -8,12 +8,14 @@
 use base Yam::Agama::patch_agama_base;
 use strict;
 use warnings;
-use testapi;
+use testapi qw(assert_script_run get_required_var select_console script_run);
 
 sub run {
     select_console 'root-console';
     my ($repo, $branch) = split /#/, get_required_var('YUPDATE_GIT');
+    script_run("dmesg --console-off");
```

Comment:
you don't need it here, we use that to not pollute the screen when agama is doing stuff behind the screen, but in this case all the out is related to the command we send.